public/js/clientLIst.js

```diff
@@ -0,0 +1,71 @@
+const tableContent = document.getElementById('clients-table-content');
+
```

Comment:
rename this file to `public/js/clientList.js` (The second letter of `List` is capitalized and should not)